vendor/engines/saml_authentication/spec/controllers/saml_authentication/sessions_controller_spec.rb

```diff
@@ -24,6 +24,17 @@
     end
 
     describe "the user does not exist already" do
+      around :each do |example|
```

Comment:
@jhanggi maybe you can help me debug--something weird is going on here. When I have `create_user: false` set locally, the specs will fail, even though if I throw a pry binding into the spec the around wrapper seems to be working properly, because inside the example it shows `Settings.saml.create_user` as `true`. I can't figure out what the issue is--any ideas?

Reply:
The `config.saml_create_user = saml_create_user?` only gets called at initialization, so changing the `Settings` value here won't affect the actual devise configuration. I think if you do `original_value = Devise.saml_create_user`. And since that already should be a boolean (since you did the nil check in the initializer), you shouldn't need the nil check here.